### ISSUE TITLE
Fix imports

### ISSUE THREAD

**Issue by benjamin-cartereau** [bdamer/drools-netbeans-plugin#2]
Hello, 
Would it be possible (too complex?) to add the "Fix Imports" functionnality with "drl" files?
Best regards

**Comment by bdamer** [comment]
Agreed that would be very useful. I'll need to review what Netbeans does to fix imports for Java.
